src/request.ts

```diff
@@ -959,6 +953,7 @@ class DatastoreRequest {
    *   const apiResponse = data[0];
    * });
    */
+  save(entities: Entities): void;
```

Comment:
I don't think this is right.  We want `save` to return a `Promise<CommitResponse>` here.  First issue is the `void`, second is the array thing pointed out above.

Reply:
🤦‍♂️ gotcha, sorry about that